presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionTask.java

```diff
@@ -93,16 +95,17 @@ public NativeExecutionTask(
                 location,
                 taskInfoCodec,
                 planFragmentCodec,
-                taskUpdateRequestCodec);
+                taskUpdateRequestCodec,
+                taskManagerConfig.getInfoRefreshMaxWait());
```

Comment:
`requireNotNull(taskManagerConfig, "taskManagerConfig is null").getInfoRefreshMaxWait()`